README.md

```diff
@@ -14,70 +14,173 @@ Takeaway Challenge
+```
 
+```
 As a customer
 So that I am reassured that my order will be delivered on time
 I would like to receive a text such as "Thank you! Your order was placed and will be delivered before 18:52" after I have ordered
 ```
 
-* Hints on functionality to implement:
-  * Ensure you have a list of dishes with prices
-  * The text should state that the order was placed successfully and that it will be delivered 1 hour from now, e.g. "Thank you! Your order was placed and will be delivered before 18:52".
-  * The text sending functionality should be implemented using Twilio API. You'll need to register for it. It’s free.
-  * Use the twilio-ruby gem to access the API
-  * Use the Gemfile to manage your gems
-  * Make sure that your Takeaway is thoroughly tested and that you use mocks and/or stubs, as necessary to not to send texts when your tests are run
-  * However, if your Takeaway is loaded into IRB and the order is placed, the text should actually be sent
-  * Note that you can only send texts in the same country as you have your account. I.e. if you have a UK account you can only send to UK numbers.
+## My Approach
+-------
 
-* Advanced! (have a go if you're feeling adventurous):
-  * Implement the ability to place orders via text message.
+To start with, I created a menu_spec and test drove the functionality for browsing a menu as per the first user story:
 
-* A free account on Twilio will only allow you to send texts to "verified" numbers. Use your mobile phone number, don't worry about the customer's mobile phone.
+```
+jonnyabrams@JONNYs-MacBook-Pro takeaway-challenge % irb -r './lib/menu'
+3.0.2 :001 > menu = Menu.new
+ => 
+#<Menu:0x0000000142857058
+... 
+3.0.2 :002 > menu.browse
+bolognese: 7.5
+carbonara: 6.5
+puttanesca: 5.5
+arrabiata: 4.5
+ => {"bolognese"=>7.5, "carbonara"=>6.5, "puttanesca"=>5.5, "arrabiata"=>4.5} 
+```
+
+Then I test-drove an Order class as my primary interface, initializing it at first with an empty array as a basket to add orders to. I later changed this to a hash to facilitate my remove_dish method, which was growing cumbersome with the array.
 
-> :warning: **WARNING:** think twice before you push your **mobile number** or **Twilio API Key** to a public space like GitHub :eyes:
->
-> :key: Now is a great time to think about security and how you can keep your private information secret. You might want to explore environment variables.
+Following the second user story, I gave my select_dish method two parameters: one for the dish itself, one for the quantity:
+
+```
+jonnyabrams@JONNYs-MacBook-Pro takeaway-challenge % irb -r './lib/order'
+3.0.2 :001 > order = Order.new
+ => 
+#<Order:0x00000001563709b8
+... 
+3.0.2 :002 > order.select_dish('bolognese', 2)
+ => 2 
+3.0.2 :003 > order.select_dish('puttanesca')
+ => 1 
+3.0.2 :004 > order.basket
+ => [["bolognese", 7.5], ["bolognese", 7.5], ["puttanesca", 5.5]] 
```

Comment:
As you mentioned in your approach, this is an ideal data set to transform into a hash